websites/docs-smoke-test/src/content/components/ai-assistant.mdx

```diff
@@ -1,5 +1,8 @@
 ---
 title: Ai Assistant
+products:
+  - aiAssistant
+  - additionalProduct
```

Comment:
can't it be like the planTags property ? `planTags: [plan1, plan2]`

just for consistency

Reply:
yes, that's right, better be consistent